bmq-sdk/src/main/java/com/bloomberg/bmq/impl/infr/util/expressionvalidator/ExpressionValidator.java

```diff
@@ -20,24 +20,22 @@
 public class ExpressionValidator {
```

Comment:
Now it's a pure utility class, so let's make its constructor private with exception throwing upon construction attempt

Like here:

https://github.com/bloomberg/blazingmq-sdk-java/blob/fa7723ddceef5d8f8c594a527d141a0afb0f52ff/bmq-sdk/src/main/java/com/bloomberg/bmq/impl/infr/codec/JsonEncoderUtil.java#L39-L41

Reply:
Made constructor `private`